src/backend/distributed/sql/citus--9.3-2--9.4-1.sql

```diff
@@ -0,0 +1,3 @@
+/* citus--9.3-2--9.4-1 */
```

Comment:
I really don't like the block comments in these migration files.

We will run this through `cpp` and remove the comments. In case there are no lines, the file will end up empty and `debuild` or `rpmbuild` will complain as the package will contain an empty file.

You do not really need to update them here, just noting this so that we update the scripts too use `--` style comments that will not be stripped out

Reply:
addressed